src/main/java/de/teamlapen/vampirism/client/core/ModKeys.java

```diff
@@ -140,6 +142,8 @@ public void handleInputEvent(InputEvent event) {
         } else if (keyPressed == KEY.ACTION2) {
             FactionPlayerHandler factionHandler = FactionPlayerHandler.get(Minecraft.getMinecraft().player);
             toggleBoundAction(factionHandler.getCurrentFactionPlayer(), factionHandler.getBoundAction2());
+        } else if (!isKeyDown(getKeyCode(KEY.SUCK))) {
```

Comment:
This event is trigger for any kind of input even a little mouse movement, which leads to dozens of EndSuckBlood messsages being sent per second. 
I would suggest setting a simple flag if the key is pressed (and the suck blood packet is sent) and then checking/clearing that flag here

Reply:
I added a variable to store if the key is held down or not.  It'll now only trigger if the suck key is being released.